### PR TITLE
Convert Chip to live code examples

### PR DESCRIPTION
**Pull request by manojleaton**
Fixes #BLUI-5787 .

#### Changes proposed in this Pull Request:

-Convert images to live RN code (using react-native-web)

-Remove unused image assets

-Convert markdown code blocks to PrismJS code blocks

-Use PrismJS to highlight targeted lines in the code example blocks

-Include buttons to copy to clipboard and view full github source

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

-

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Comment by JeffGreiner-eaton** [comment]
its throwing error here for some reason

image

File: docs/src/componentDocs/Chip/examples/Chip.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {Box} from '@mui/material';
+import {ChipExample} from './ChipExample';
+import {CodeBlock, CodeBlockActionButtonRow} from '../../../shared';
+
+const codeSnippet = `<Chip>Outlined Chip</Chip>
```
Comment (ektaghag-eaton) [inline docs/src/componentDocs/Chip/examples/Chip.tsx:6]: format this code properly

File: package.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
   },
   "private": true,
   "dependencies": {
+    "@brightlayer-ui/colors": "^4.0.0",
```
Comment (JeffGreiner-eaton) [inline package.json:14]: why are we trying to use colors version 4?

File: docs/src/componentDocs/Chip/examples/Chip.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {Box} from '@mui/material';
+import {ChipExample} from './ChipExample';
+import {CodeBlock, CodeBlockActionButtonRow} from '../../../shared';
+
+const codeSnippet = `
+<Chip>Outlined Chip</Chip>
```
Comment (surajeaton) [inline docs/src/componentDocs/Chip/examples/Chip.tsx:7]: Snippets are adding an extra blank line at the top, please move `<Chip>Outlined Chip</Chip>` at the first line

File: docs/src/componentDocs/Chip/markdown/ChipExamples.mdx [hunk 4]
```diff
@@ -1,93 +1,22 @@
...
-import ChipAvatarExample from '../images/ChipAvatarExample.png';
-import ChipSelectedExample from '../images/ChipSelectedExample.png';
-import ChipDisableExample from '../images/ChipDisableExample.png';
+import {Chip} from '../examples/Chip';
+import {ChipWithIcon} from '../examples/ChipWithIcon';
+import {ChipSelect} from '../examples/ChipSelect';
 
 ## Chip
 
 The basic chip is a clickable element displaying a text label.
```
Comment (surajeaton) [inline docs/src/componentDocs/Chip/markdown/ChipExamples.mdx:8]: Please add description from https://brightlayer-ui-components.github.io/react-native/?path=/info/components-documentation--chip

**Review by surajeaton — APPROVED** [review]

**Pull request merged**